<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1449</width>
    <height>703</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="GRAPH" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="TABLE">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>11</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>x i-1 </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x i</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>a i</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>b i</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c i</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>d i</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="BUTTON">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_CLEAN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Parametrs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>194</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Parametrs</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0">
      <widget class="QLabel" name="label_k">
       <property name="text">
        <string>k =</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_n_1">
       <property name="text">
        <string>n =</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_Sb">
       <property name="text">
        <string>S''[b] =</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_Sa">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_Sa">
       <property name="text">
        <string>S''[a] =</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_Sb">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_a">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_a">
       <property name="text">
        <string>a =</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_b">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="lineEdit_n">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_b">
       <property name="text">
        <string>b =</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="lineEdit_k">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBox_Auto">
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>281</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Information</string>
    </property>
    <widget class="QLabel" name="label_n">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>n = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxFS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max|F-S| =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_N">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>N =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxFSX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxFS_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max|F'-S'| =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxFSX_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxFS__">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max|F&quot;-S&quot;| =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxFSX__">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x =</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="TABLE_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>801</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|F-S|</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F'</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S'</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|F'-S'|</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F&quot;</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S&quot;</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|F&quot;-S&quot;|</string>
     </property>
    </column>
   </widget>
   <widget class="QCustomPlot" name="GRAPH_ERROR" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="GRAPH_DER" native="true">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="GRAPH_DER_ERROR" native="true">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_TEST">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TEST</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_FIRST">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x^1/2*cos(x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_SECOND">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+ cos(10x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_THIRD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+ cos(100x)</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="GRAPH_DER_DER" native="true">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="GRAPH_DER_DER_ERROR" native="true">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>220</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1449</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBox_Auto</tabstop>
  <tabstop>lineEdit_a</tabstop>
  <tabstop>lineEdit_b</tabstop>
  <tabstop>lineEdit_Sa</tabstop>
  <tabstop>lineEdit_Sb</tabstop>
  <tabstop>lineEdit_n</tabstop>
  <tabstop>lineEdit_k</tabstop>
  <tabstop>radioButton_SECOND</tabstop>
  <tabstop>radioButton_THIRD</tabstop>
  <tabstop>TABLE</tabstop>
  <tabstop>radioButton_TEST</tabstop>
  <tabstop>TABLE_1</tabstop>
  <tabstop>radioButton_FIRST</tabstop>
  <tabstop>BUTTON</tabstop>
  <tabstop>BUTTON_CLEAN</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
